mne/utils/docs.py

```diff
@@ -2295,7 +2295,7 @@
     respective epoch will be dropped.
 
     The dictionary keys correspond to the different channel types; valid
-    keys are: ``'grad'``, ``'mag'``, ``'eeg'``, ``'eog'``, and ``'ecg'``.
+    keys include ``'grad'``, ``'mag'``, ``'eeg'``, ``'eog'``, and ``'ecg'``.
```

Comment:
I think the phrasing in the other docstring is better, let's standardize them:
```suggestion
    **keys** can be any channel type present in the object.
```

Reply:
Shouldn't it be plural ("channel type**S**")? Asking for a friend whose first language isn't American 😅

Reply:
The quantifier "any" takes a singular noun. (consider: "anyone", not "anyones" 🙂)